configs/tidyverse.json

```diff
@@ -19,14 +18,6 @@
     ".dont-index"
   ],
   "sitemap_urls": [
-    "https://dplyr.tidyverse.orgsitemap.xml"
-  ],
-  "custom_settings": {
-    "separatorsToIndex": "_"
```

Comment:
Are you sure you want this settings to be removed?

Reply:
@jayhesselberth — is this one a must? (Thanks for the feedback @s-pace)

Reply:
I would recommend you to keep it in order to be able to search for argument such like `group_by_prepare`. Without it, the previous query will not fully match this argument because of [algolia normalisation.](https://www.algolia.com/doc/guides/managing-results/optimize-search-results/handling-natural-languages-nlp/?language=php#how-the-engine-normalizes-data)